package.json

```diff
@@ -7,10 +7,8 @@
     "action:list": "npm run action list",
     "action": "node ./infrastructure/build/run_action.mjs",
     "clean": "npm run clean --workspaces && rm -rf output node_modules third_party/jsign/*.jar && go run github.com/go-task/task/v3/cmd/task clean",
-    "format:all": "prettier --write \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
-    "format": "pretty-quick --staged --pattern \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
-    "lint:ts": "eslint --ext ts,mjs client",
-    "lint": "npm run lint:ts",
+    "format": "gts fix",
```

Comment:
Are these also only for staged files?

Reply:
I'm just wondering if there's a way to specify only changed files here, otherwise it's hard to format only your changed files when developing. But if there's no way to specify that, it's fine.